src/index.js

```diff
@@ -0,0 +1,80 @@
+import './components/addremove/style.scss';
+import UI from './components/addremove/AddRemoveClass.js';
+import Checkbox from './components/checkbox/CheckboxClass.js';
+
+const reloadWindow = document.querySelector('.fa-arrows-rotate');
+
+// Event Listener to render tasks when window loads.
+window.addEventListener('DOMContentLoaded', UI.renderTasks);
+
+// Event Listeners
```

Comment:
Please do not leave unused comments, as they make the code verbose and untidy.

Reply:
👌